specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/preview/2015-11-01-preview/examples/ManagementConfigurationCreate.json

```diff
@@ -5,51 +5,7 @@
         "api-version":"2015-11-01-preview",
         "subscriptionId":"subid",
         "parameters":{
-            "location":"East US",
-            "properties":{
-                "applicationId":"/subscriptions/sub1/resourcegroups/rg1/providers/Microsoft.Appliance/Appliances/appliance1",
-                "parentResourceType":"Microsoft.OperationalInsights/workspaces",
-                "template": {
-                        "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
-                        "contentVersion": "1.0.0.0",
-                        "parameters": {
-                            "parentResourceName": {
-                                "type": "string"
-                            },
-                            "managementConfigurationName": {
-                                "type": "string"
-                            },
-                            "jsonobject": {
-                                "type": "object"
-                            }
-                        },
-                        "resources": [
-                            {
-                                "apiVersion": "2015-11-01-preview",
-                                "name": "[concat(parameters('parentResourceName'), '/', parameters('managementConfigurationName'), '-', parameters('jsonobject').displayName)]",
-                                "type": "Microsoft.OperationalInsights/workspaces/savedsearches",
-                                "dependsOn": [],
-                                "properties": {
-                                    "ETag": "*",
-                                    "Category": "A-Templated",
-                                    "DisplayName": "[parameters('jsonobject').displayName]",
-                                    "Query": "[parameters('jsonobject').query]",
-                                    "Version": 1
-                                }
-                            }
-                        ],
-                        "outputs": {}
-                },                
-                "parameters": [
-                        {
-                            "name": "jsonobject",
-                            "value": {
-                                "displayName": "abcde",
-                                "query": "hello"
-                            }
-                        }
-                ]
-            }
+            "location":"East US"
```

Comment:
This create requires no parameters?

Reply:
These are non-used API's, created before a re-org. We plan to remove them after doing a little more diligence.
So took quickest rout in order to get the test passing.